### DISCUSSION TITLE
Prettier path and yarn plug and play (PnP)

### DISCUSSION THREAD

**Discussion by torkelo** [LunarVim/LunarVim#1901] [Q&A] [answered]
Prettier formatter is not working when I work in repositories that use yarm PnP (plug and play), https://yarnpkg.com/features/pnp 

Is there a way to change path to prettier or have it run through `yarn prettier`  ?

**Comment by chase** [comment]
Have you tried installing it globally without using yarn? `npm i -g prettier`

**Reply by torkelo** [reply]
does not help

**Comment by LostInTheLogs** [comment, marked as answer]
https://github.com/LunarVim/LunarVim/issues/3482
